### ISSUE TITLE
Can we use ssd_inception_v3_pets.config for object detection training

### ISSUE THREAD

**Issue by aliBUT** [tensorflow/models#5314]
I am new to Tensorflow but I understand that training for image classification and object detection are two different processes. I am trying to do training for Object detection and I was planing to use ssd_inception_v3_pets.config but then in the files I saw this line `loss_type: CLASSIFICATION` does that mean this config is for classification training only? 

Please let me know 

thanks

**Comment by netanel-s** [comment]
You plan to train on which dataset? A public one? or maybe your own?
In the configs folder you can find several datasets, such as pets and coco.
The `loss_type` you're referring to is for the `hard_example_miner` - what examples are considered hard? those with the high classification loss.
The loss of the entire OD CNN is comprised of both `localization_loss` and `classification_loss`, which are also present in the same config file.

**Comment by aliBUT** [comment]
hi @netanel-s 

I have my own data set of 11 items you can find in kitchen and fridge. I prepared my dataset pictures using labelimg.  so can this same ssd_inception_v3_pets.config be used for object detection training?

**Comment by aliBUT** [comment]
ok thanks . one more quick question whats the difference b/w pets and COCO ?

**Comment by netanel-s** [comment]
The categories included in the dataset, and therefore its content.
[Here](http://www.robots.ox.ac.uk/~vgg/data/pets/) you can see Pets dataset, and in [here](http://cocodataset.org/#explore) COCO.
